app/assets/javascripts/discourse/app/components/site-header.js

```diff
@@ -276,6 +253,27 @@ const SiteHeaderComponent = MountWidget.extend(
       });
     },
 
+    _stickyHeaderCheck() {
+      const dockAnchor = document.querySelector(".header-dock-anchor");
+      if (dockAnchor) {
+        this._dockAnchor = dockAnchor;
+
+        const dockObserver = new IntersectionObserver((entries) => {
+          if (!entries[0].isIntersecting) {
+            document.body.classList.add("docked");
+            this.dockedHeader = true;
```

Comment:
where is this used now ?

Reply:
in any case it should have be _dockedHeader, doesn’t seem like it's used in public API

Reply:
It's not used anymore, so I'll just remove it for now. I checked all-the-themes, all-the-plugins, all-custom-themes, and all-custom-plugins repos and it's not used there either.